src/goSignature.ts

```diff
@@ -43,7 +43,7 @@ export class GoSignatureHelpProvider implements SignatureHelpProvider {
 				let funcName = declarationText.substring(0, nameEnd);
 				sig = declarationText.substring(sigStart);
 				si = new SignatureInformation(funcName + sig, res.doc);
-			} else {
+			} else if (res.toolUsed === 'gogetdoc') {
```

Comment:
Both Signature Help and Hover Info features are currently broken.
Until `guru` has the feature to provide docs, you can call `godoc` to provide the docs.

Pull https://github.com/Microsoft/vscode-go/blob/1302fcf0f704f66c974e043258b7ed9783ba104b/src/goDeclaration.ts#L85-L105 out into a function and use it in both `definitionLocation_godef` and `definitionLocation_guru`